src/kvstore/KVStore.h

```diff
@@ -13,11 +13,20 @@
 #include "kvstore/Common.h"
 #include "kvstore/KVIterator.h"
 #include "kvstore/PartManager.h"
+#include "meta/SchemaManager.h"
 
 namespace nebula {
 namespace kvstore {
 
 struct KVOptions {
+    /**
+     * HBase thrift server address.
+     * */
+    HostAddr hbaseServer_;
```

Comment:
move it into `HBaseStore.h` 
PS the HBase relation code moved to a new directory would be better ?

Reply:
Yep, my basic principle for implementing this PR is to try not to change the existing directory structure. I am worried that it is not reasonable. Now, I will adopt your new proposal. thx.